commands/utils/params.go

```diff
@@ -150,7 +150,7 @@ func extractVcsProviderFromEnv() (vcsutils.VcsProvider, error) {
 		return vcsutils.GitHub, nil
 	case string(GitLab):
 		return vcsutils.GitLab, nil
-	case string(BitbucketServer):
+	case string(BitbucketServer), "bitbucket server":
```

Comment:
Let's explain here why we're checking the two values, so that someone don't decide to remove it in the future.